.github/workflows/ci.yml

```diff
@@ -6,23 +6,42 @@ on:
   pull_request:
     branches:
       - '**'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 jobs:
-  test:
+  check-lint:
+    name: Lint
+    timeout-minutes: 10
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - run: npm run lint
+  test:
     strategy:
       matrix:
-        node: [ '14', '16', '18' ]
+        node: [ '14', '16', '18', '19' ]
```

Comment:
We'd add a Node 19 a test if we officially add Node 19 support for the adapter. This is a feature change that would need to be done in a separate PR.

Reply:
Removed 19